process-migration/.local/pim-overrides.yaml

```diff
@@ -0,0 +1,21 @@
+schema_version: 1
```

Comment:
What is the purpose of this file? I think it should be deleted... We only use branch-overrides.yaml and tag-overrides.yaml.

Reply:
Argh!! I added it by mistake. I used it for my local tests

Reply:
Solved

Reply:
@ruromero No worries. Just FYI, any file in these repos that start with "my" gets ignored by git (they're in the .gitignore file), so if you have yaml for your own local purposes in the future, I'd suggest that.